### PR TITLE
Feature/metadata

### PR DESCRIPTION
**Pull request by jmalczyk**
Some metadata cleanup. Uses dataset_id to find dataset and source specific metadata instead of  provider, type, and class.  Drops layer specific metadata queries.

### CODE REVIEW

File: app/static/js/mol.js [hunk 1]
```diff
@@ -4837,236 +5028,199 @@ mol.modules.map.metadata = function(mol) {
...
+                        ).get())+
+                    Math.max.apply(
+                        Math, 
+                        $(self).find('.values').map(
+                            function(){ 
+                                return $(this).width() 
+                            }
+                        ).get())+150
+            );
+            //this.dialog.moveToTop();
```
Comment (eightysteele) [inline app/static/js/mol.js:5220]: If we don't need these lines, let's remove them. Otherwise maybe add a comment for why we're commenting it out here.

**Comment by eightysteele** [comment]
LGTM. Minor commenting nit. Otherwise #shipit.

**Comment by jmalczyk** [comment]
We actually should still fire moveToTop(). Put it back in. Shipping.

**Comment by eightysteele** [comment]
Boom.

**Pull request merged**